frontend/src/component/context/ContextFieldUsage/ContextFieldUsage.tsx

```diff
@@ -0,0 +1,89 @@
+import { Alert, styled } from '@mui/material';
+import { formatEditStrategyPath } from 'component/feature/FeatureStrategy/FeatureStrategyEdit/FeatureStrategyEdit';
+import { IFeatureStrategy } from 'interfaces/strategy';
+import { Link } from 'react-router-dom';
+import { formatStrategyName } from 'utils/strategyNames';
+import { useStrategiesByContext } from 'hooks/api/getters/useStrategiesByContext/useStrategiesByContext';
+import useProjects from 'hooks/api/getters/useProjects/useProjects';
+
+const StyledUl = styled('ul')(({ theme }) => ({
+    listStyle: 'none',
+    paddingLeft: 0,
+}));
+
+const StyledAlert = styled(Alert)(({ theme }) => ({
+    marginTop: theme.spacing(1),
+}));
+
+interface IContextFieldUsageProps {
+    contextName: string;
+}
+
+export const ContextFieldUsage = ({ contextName }: IContextFieldUsageProps) => {
+    const { strategies } = useStrategiesByContext(contextName);
+    const { projects } = useProjects();
+
+    const projectsUsed = Array.from(
+        new Set<string>(
+            strategies.map(({ projectId }) => projectId!).filter(Boolean)
+        )
+    );
+
+    const projectList = (
+        <StyledUl>
+            {projectsUsed.map(projectId => (
+                <li key={projectId}>
+                    <Link
+                        to={`/projects/${projectId}`}
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        {projects.find(({ id }) => id === projectId)?.name ??
+                            projectId}
+                    </Link>
+                    <ul>
+                        {strategies
```

Comment:
I'd prepare projectStrategies variable outside of JSX to minimize scripting inside HTML tags. I'd only use map operation in JSX, not filter